content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Create a Windows HostProcess Pod
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+Windows HostProcess containers enable you to run containerized 
+workloads on a Windows host. These containers operate as 
+normal processes but have access to the host network namespace, 
+storage, and devices when given the appropriate user privileges. 
+HostProcess containers can be used to deploy CNIs, 
+storage configurations, device plugins, kube-proxy, and other 
+components to Windows nodes without the need for dedicated proxies or 
+the direct installation of host services.
+
+Administrative tasks such as installation of security patches, event 
+log collection, and more can be performed without requiring cluster operators to 
+log onto each Window node. HostProcess containers can run as any user that is 
+available on the host or is in the domain of the host machine, allowing administrators 
+to restrict resource access through user permissions. While neither filesystem or process 
+isolation are supported, a new volume is created on the host upon starting the container 
+to give it a clean and consolidated workspace. HostProcess containers can also be built on 
+top of existing Windows base images and do not inherit the same 
+[compatibility requirements](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility) 
+as Windows server containers, meaning that the version of the base images does not need 
+to match that of the host.
+
+### When should I use a Windows HostProcess container?
+
+- When you need to perform tasks which require the networking namespace of the host. 
+HostProcess containers have access to the host's network interfaces and IP.
```

Comment:
nit:
```suggestion
HostProcess containers have access to the host's network interfaces and IP addresses.
```
Last time I used Windows server I think it enabled both IPv4 and IPv6 by default, so it's normal to find a host with more than one IP address.